tools/reviewer/service/tests/CodeReviewServiceTextDiffTest.java

```diff
@@ -174,11 +174,10 @@ private void createInitialRepo(String initialRepoFolder) {
 
   private void initAaBase(String initialRepoFolder, String aaBaseFolder) {
     InitCommand initCommand = component.getInitCommand();
-    InitCommand.basePath.resetValueForTesting();
     InitCommand.startuposRepo.resetValueForTesting();
     String[] args = {
+      "init", aaBaseFolder,
```

Comment:
If we removed `--base_path`, why do we need to add `init`?

Reply:
It happens because we changed the logic of parsing input args.
We used flags before:
- `bazel run //tools/aa:aa_tool -- init --base_path </path/to/base/folder> --startupos_repo <url_to_startupos_repo>`
- `aa init --base_path </path/to/base/folder> --startupos_repo <url_to_startupos_repo>`
And if we use `init` command directly (`initCommand.run(args)`) we could just to supply flags in an arbitrary manner.

Now, init command looks like:
- `bazel run //tools/aa:aa_tool -- init </path/to/base/folder> --startupos_repo <url_to_startupos_repo>`
- `aa init </path/to/base/folder> --startupos_repo <url_to_startupos_repo>`
Now, input args is parsed otherwise. We get </path/to/base/folder> by a specific index (args[1] or args[2]). And if <path/to/base/folder> will be not in the right place it won't be working.